.github/workflows/e2e-rollup-workflow.yml

```diff
@@ -0,0 +1,124 @@
+        echo "import square from './square.js'; export default function cube(x) { return square(x) * x; }" | tee src/cube.js
+        echo "import cube from './cube.js'; export default function hyperCube(x) { return cube(x) * x; }" | tee src/hyperCube.js
+        echo "export default function square(x) { return x * x; }" | tee src/square.js
+
+        yarn rollup -c
+        [[ "$(node dist/main.js)" = "625" ]]
+        [[ "$(node dist/otherEntry.js)" = "125" ]]
+        ls dist | grep "cube"
+
+        rm -rf dist src
+
+        # With NPM packages
+        yarn add the-answer@^1.0.0
+        yarn add -D @rollup/plugin-node-resolve@^7.0.0
+
+        echo "import resolve from '@rollup/plugin-node-resolve'; export default { input: 'src/main.js', output: { file: 'dist/bundle.js', format: 'cjs' }, plugins: [resolve()]};" | tee rollup.config.js
+
+        mkdir src
+        echo "import answer from 'the-answer'; console.log('the answer is ' + answer);" | tee src/main.js
+
+        yarn rollup -c
+        [[ "$(node dist/bundle.js)" = "the answer is 42" ]]
+
+        rm -rf dist src
+
+        # Peer dependencies
+        yarn add -P lodash
+
+        echo "import resolve from '@rollup/plugin-node-resolve'; export default { input: 'src/main.js', output: { file: 'dist/bundle.js', format: 'cjs' }, plugins: [resolve()], external: ['lodash']};" | tee rollup.config.js
+
+        mkdir src
+        echo "import answer from 'the-answer'; import _ from 'lodash';" | tee src/main.js
+
+        yarn rollup -c
+        cat dist/bundle.js | grep -v "lodash"
+
+        rm -rf src dist
+
+        # Babel
+        yarn add -D rollup-plugin-babel@^4.3.3 @babel/core@^7.7.7 @babel/preset-env@^7.7.7
+
+        echo "import resolve from '@rollup/plugin-node-resolve'; import babel from 'rollup-plugin-babel'; export default { input: 'src/main.js', output: { file: 'dist/bundle.js', format: 'cjs' }, plugins: [resolve(), babel({ exclude: 'node_modules/**' })]};" | tee rollup.config.js
+
+        mkdir src
+        echo '{ "presets": [["@babel/preset-env", { "modules": false }]] }' | tee src/.babelrc
+        echo "import answer from 'the-answer'; console.log(\`the answer is \${answer}\`);" | tee src/main.js
+
+        yarn rollup -c
+        [[ "$(node dist/bundle.js)" = "the answer is 42" ]]
+        cat dist/bundle.js | grep -v "console.log(\`"
+
+        rm -rf src dist
+
+        # rollup-plugin-postcss
+        yarn add -D rollup-plugin-postcss@^2.0.3
+
+        echo "import postcss from 'rollup-plugin-postcss'; export default { input: 'src/main.js', output: { file: 'dist/bundle.js', format: 'cjs' }, plugins: [postcss({ extract: true, modules: true })] }" | tee rollup.config.js
+
+        mkdir src
+        echo "import style from './style.css'; console.log(style);" | tee src/main.js
+        echo ".app { color: red; }" | tee src/style.css
+
+        yarn rollup -c
+        [[ "$(cat dist/bundle.css)" = ".style_app__3FC6W { color: red; }" ]]
```

Comment:
Is the `3FC6W` stable? Can it cause the build to periodically fail?

Reply:
Since it's a hash of the css file, which means that if we don't change the content of the css file, and the hash function never changed, it would be fine.

However, rollup also do not guarantee that the hashing function would never changed, we could argue that the result is not stable... Maybe we should think of other alternatives, any idea?

Reply:
Typically this can be solved by doing a replace operation: `| sed 's/__[0-9A-F]+/__xxx/'`. This way the result is always stable.

Reply:
Maybe a regex approach with `=~` would be much cleaner 🤔? I can open up a following PR if that sounds good to you 😉